mercury/model/resp.go

```diff
@@ -0,0 +1,177 @@
+package model
+
+import (
+	"github.com/nervosnetwork/ckb-sdk-go/types"
+	"math/big"
+)
+
+type IoType string
+type DaoStateType = string
+type DBDriver string
+type NetworkType string
+type SyncState string
+type AccountType string
+type TxViewType string
+
+const (
+	IoTypeInput          IoType       = "Input"
+	IoTypeOutput         IoType       = "Output"
+	DaoStateTypeDeposit  DaoStateType = "Deposit"
+	DaoStateTypeWithdraw DaoStateType = "Withdraw"
+
+	AccountTypeAcp    AccountType = "Acp"
+	AccountTypePwLock AccountType = "PwLock"
+
+	DBDriverPostgreSQL DBDriver = "PostgreSQL"
+	DBDriverMySQL      DBDriver = "MySQL"
+	DBDriverSQLite     DBDriver = "SQLite"
+
+	NetworkTypeMainnet NetworkType = "Mainnet"
+	NetworkTypeTestnet NetworkType = "Testnet"
+	NetworkTypeStaging NetworkType = "Staging"
+	NetworkTypeDev     NetworkType = "Dev"
+
+	SyncStateReadOnly            SyncState = "ReadOnly"
+	SyncStateSerial              SyncState = "Serial"
+	SyncStateParallelFirstStage  SyncState = "ParallelFirstStage"
+	SyncStateParallelSecondStage SyncState = "ParallelSecondStage"
+
+	TxViewTypeTransactionWithRichStatus TxViewType = "TransactionWithRichStatus"
+	TxViewTypeTransactionInfo           TxViewType = "TransactionInfo"
+)
+
+type Balance struct {
+	Ownership string     `json:"ownership"`
+	AssetInfo *AssetInfo `json:"asset_info"`
+	Free      *big.Int   `json:"free"`
+	Occupied  *big.Int   `json:"occupied"`
+	Frozen    *big.Int   `json:"frozen"`
+}
+
+type GetBalanceResponse struct {
+	Balances       []*Balance `json:"balances"`
+	TipBlockNumber uint64     `json:"tip_block_number"`
+}
+
+type DBInfo struct {
+	Version   string   `json:"version"`
+	DB        DBDriver `json:"db"`
+	ConnSize  uint32   `json:"conn_size"`
+	CenterId  int64    `json:"center_id"`
+	MachineId int64    `json:"machine_id"`
+}
+
+type AccountInfo struct {
+	AccountNumber  uint32      `json:"account_number"`
+	AccountAddress string      `json:"account_address"`
+	AccountType    AccountType `json:"account_type"`
+}
+
+type BlockInfo struct {
+	BlockNumber  uint64             `json:"block_number"`
+	BlockHash    types.Hash         `json:"block_hash"`
+	ParentHash   types.Hash         `json:"parent_hash"`
+	Timestamp    uint64             `json:"timestamp"`
+	Transactions []*TransactionInfo `json:"transactions"`
+}
+
+type TransactionInfoWrapper struct {
+	Type  TxViewType       `json:"type"`
+	Value *TransactionInfo `json:"value"`
+}
+
+type TransactionViewWrapper struct {
+	Type  TxViewType                 `json:"type"`
+	Value *TransactionWithRichStatus `json:"value"`
+}
```

Comment:
according to naming convention: `TransactionInfo` => `TransactionInfoWrapper` , `TransactionWithRichStatus` should be `TransactionWithRichStatusWrapper`